sdk/cosmos/azure-cosmos/azure/cosmos/aio/_container.py

```diff
@@ -707,10 +729,14 @@ async def patch_item(
         :keyword Literal["High", "Low"] priority: Priority based execution allows users to set a priority for each
             request. Once the user has reached their provisioned throughput, low priority requests are throttled
             before high priority requests start getting throttled. Feature must first be enabled at the account level.
-        :returns: A dict representing the item after the patch operations went through.
+        :keyword bool response_payload_on_write_disabled: Indicates whether service should be instructed to skip
+            sending response payloads. When not specified explicitly here, the default value will be determined from 
+            kwargs or when also not specified there from client-level kwargs.  
```

Comment:
That is why I kept the name on client level as response_payload_on_write_disabled - it would be applicable for all write operations. Change the name of per operation kwargs to no_reponse like proposed below.